src/test/java/com/example/springtweet/springtweet/controller/TwitterControllerTest.java

```diff
@@ -0,0 +1,498 @@
+                    }
+
+                    @Override
+                    public int compareTo(User o) {
+                        return 0;
+                    }
+
+                    @Override
+                    public RateLimitStatus getRateLimitStatus() {
+                        return null;
+                    }
+
+                    @Override
+                    public int getAccessLevel() {
+                        return 0;
+                    }
+                };
+            }
+
+            @Override
+            public boolean isRetweet() {
+                return false;
+            }
+
+            @Override
+            public Status getRetweetedStatus() {
+                return null;
+            }
+
+            @Override
+            public long[] getContributors() {
+                return new long[0];
+            }
+
+            @Override
+            public int getRetweetCount() {
+                return 0;
+            }
+
+            @Override
+            public boolean isRetweetedByMe() {
+                return false;
+            }
+
+            @Override
+            public long getCurrentUserRetweetId() {
+                return 0;
+            }
+
+            @Override
+            public boolean isPossiblySensitive() {
+                return false;
+            }
+
+            @Override
+            public String getLang() {
+                return null;
+            }
+
+            @Override
+            public Scopes getScopes() {
+                return null;
+            }
+
+            @Override
+            public String[] getWithheldInCountries() {
+                return new String[0];
+            }
+
+            @Override
+            public long getQuotedStatusId() {
+                return 0;
+            }
+
+            @Override
+            public Status getQuotedStatus() {
+                return null;
+            }
+
+            @Override
+            public int compareTo(Status o) {
+                return 0;
+            }
+
+            @Override
+            public UserMentionEntity[] getUserMentionEntities() {
+                return new UserMentionEntity[0];
+            }
+
+            @Override
+            public URLEntity[] getURLEntities() {
+                return new URLEntity[0];
+            }
+
+            @Override
+            public HashtagEntity[] getHashtagEntities() {
+                return new HashtagEntity[0];
+            }
+
+            @Override
+            public MediaEntity[] getMediaEntities() {
+                return new MediaEntity[0];
+            }
+
+            @Override
+            public SymbolEntity[] getSymbolEntities() {
+                return new SymbolEntity[0];
+            }
+
+            @Override
+            public RateLimitStatus getRateLimitStatus() {
+                return null;
+            }
+
+            @Override
+            public int getAccessLevel() {
+                return 0;
+            }
+        };
```

Comment:
Why have you implemented all these methods for the new object you are creating over here? You need to mock the status class here instead of creating a new object with all the dummy methods implemented. 
**_Status status1 = Mockito.mock(Status.class);_** should work